frontend/package-lock.json

```diff
@@ -17427,17 +17446,17 @@
       "integrity": "sha512-fTGnTG7FhtTG9tiYDL3k5s8YHqB71Ml5+aC9B7eaBygeB8GKXBrcbTXLOzoCRxT3Jr5cRhr99PMBu0AkqmhBog=="
     },
     "vega-view": {
-      "version": "5.5.0",
-      "resolved": "https://registry.npmjs.org/vega-view/-/vega-view-5.5.0.tgz",
-      "integrity": "sha512-hD3J00QRkh3zC1jF81UgxnXpTKY07nVKKG1ut7rlCpVdpEec8Ix6EckOHpAzMkHeY6aG5fbhSQbRYv05tzQy/A==",
+      "version": "5.6.0",
+      "resolved": "https://registry.npmjs.org/vega-view/-/vega-view-5.6.0.tgz",
+      "integrity": "sha512-/w3R46lw11SXGRwuhZTvhDtQCPBPf21Svh1wiEjQ/f6M1DMhJySaPhdVdh9kIAucYAQtlrBJe1U82PG9j2wAdQ==",
       "requires": {
         "d3-array": "^2.4.0",
```

Comment:
@michaelwoodruffdev and @gabe-heim and @ccarterlandis : I explicitly moved the "d3-array" and "d3-timer" from MIchael's branch into the updated version. I do not think he would have added those requirements if they were not needed, but check.

Reply:
I will defer to @michaelwoodruffdev / @gabe-heim